pkg/gardenlet/features/features.go

```diff
@@ -26,8 +26,8 @@ var (
 	FeatureGate  = featuregate.NewFeatureGate()
 	featureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 		features.Logging:              {Default: false, PreRelease: featuregate.Alpha},
-		features.HVPA:                 {Default: false, PreRelease: featuregate.Alpha},
-		features.HVPAForShootedSeed:   {Default: false, PreRelease: featuregate.Alpha},
+		features.HVPA:                 {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Isn't there already a redesign work for hvpa - ref https://github.com/gardener/hvpa-controller/pull/76? Do we want to enable it despite the fact that it will be changed/redesigned?

Reply:
@ggaurav10 @amshuman-kr can you comment on that, please?

Reply:
Yes, [gardener/hvpa-controller#76](https://github.com/gardener/hvpa-controller/pull/76) is a redesigned implementation for HVPA based on the new [design proposal](https://github.com/gardener/hvpa-controller/blob/master/docs/proposals/HVPA-EP-0001/HVPA-EP-001.md) to be able to control horizontal scaling as well as vertical scaling (currently only the vertical scaling is controlled).

The design has been reviewed and the implementation is under review. I have a few suggestions for the implementation, so it will take some time for the implementation to be merged. Also, we need to run a round of performance test to verify the changes are working fine at scale. So, it might be a few weeks before the redesign is released.

AFAIK, the HVPA feature gates are already enabled in our own landscapes. So, if the concern is that redesign might cause instability, then the feature gates themselves will not help (at least not for our own landscapes). We will need other mechanisms to mitigate (good test coverage, performance testing, phased rollout of the new version etc.)

So, I am fine promoting the feature gate. But no strong opinion. I am fine holding the promotion too.

Reply:
Then I suggest to keep it alpha, as we have done with the logging https://github.com/gardener/gardener/issues/2175#issuecomment-633915482

Reply:
Ah, there is precedence. Makes sense 👍